proto/axelar/nexus/exported/v1beta1/types.proto

```diff
@@ -82,10 +82,10 @@ enum TransferDirection {
 
   TRANSFER_DIRECTION_UNSPECIFIED = 0
       [ (gogoproto.enumvalue_customname) = "Unspecified" ];
-  TRANSFER_DIRECTION_INCOMING = 1
-      [ (gogoproto.enumvalue_customname) = "Incoming" ];
-  TRANSFER_DIRECTION_OUTGOING = 2
-      [ (gogoproto.enumvalue_customname) = "Outgoing" ];
+  TRANSFER_DIRECTION_FROM = 1
```

Comment:
we save in the keeper based on the enum values, so name change shouldn't be breaking